src/sentry/tasks/commits.py

```diff
@@ -160,7 +160,7 @@ def fetch_commits(release_id: int, user_id: int, refs, prev_release_id=None, **k
                 },
             )
             span = sentry_sdk.Hub.current.scope.span
-            span.set_status("unknown_error")
+            span.set_status("unknown_error")  # type: ignore
```

Comment:
this feels like it shouldn't be ignored -- what's the error here?  type ignores should usually be reserved for mypy bugs

Reply:
gotcha! so in this case, would it better to check that `span` is `None` and raise an error if it is, or use a try/except and push all the code afterwards in it?

Reply:
this looks like a private api -- I think `sentry_sdk.get_current_span().set_status(...)` is maybe less internal?